src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,91 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+// Increase the count after successful form submission
+// to reset touched status of all the `Field`s
+const initialValue = {
```

Comment:
This object containt many values
```suggestion
const initialValues = {
```